docs-content/installing.html.md.erb

```diff
@@ -29,16 +27,22 @@ Do the following:
 1. If the AppDynamics Platform Monitoring tile does not appear in the Installation Dashboard, click **Import a Product** in the left-side menu and import the AppDynamics Platform Monitoring tile.
 
 1. Double-click the **AppDynamics Platform Monitoring for PCF** tile in the Installation Dashboard.
-   The Settings tab is displayed.
+   A Settings tab is displayed.
 
 1. In the **Settings** tab, select **AppDynamics Nozzle Configuration**.
 ![](./images/appd-nozzle-config.png)
 
-1. Specify the following Nozzle configuration settings and click **Save**. All the fields are mandatory. You cannot add multiple Nozzle configurations.
+1. Specify the AppDynamics Nozzle configuration settings. All fields are mandatory. You cannot add multiple Nozzle configurations.
 
-  * **UAA token_endpoint URL**: The authentication token endpoint URL of the User Account and Authentication Service (UAA) API.
-  * **Doppler Traffic Controller URL**: The doppler firehose endpoint.
+    * **UAA token_endpoint URL**: The User Account and Authentication Service (UAA) API authentication token endpoint URL.
+    * **Doppler Logging Endpoint**: The doppler firehose endpoint address.
+    * **Nozzle Username**: Pivotal Cloud Foundry user name.
+    * **Nozzle Password**: Pivotal Cloud Foundry user's password.
+    * **Controller Account Name**: The name of the account in the AppDynamics Controller.
+    * **Account Access Key**: The access key for the Controller account.
+    * **Controller Host**: AppDynamics host, such as https://example.cloud.appdynamics.com:8092.
 
+1. Click **Save**.
 
     * Run the  cf command to `cf curl /v2/info`. The UAA token_endpoint and Doppler Traffic Controller URL are the values of `token_endpoint` and `doppler_logging_endpoint` respectively
```

Comment:
In this section, I am unclear as to if this is: 

- a second set of steps to do using the Cloud Foundry command line after configuring the Ops Manager via the UI? 
- or an alternate way to perform the same configuration steps using the cf CLI. 

If it is performing the same steps, we should say: Using the Ops Manager to Configure AppDynamics Nozzle and Using the cf CLI to Configure the AppDynamics Nozzle.

Reply:
This is second set of steps to figure out the values on the form fields. This needs to be done in addition to OpsManager step